Author: Luca Boccassi <luca.boccassi@gmail.com>
Description: Use put_page instead of page_cache_release
 From kernel 4.6-rc3 and newer page_cache_release was dropped. It has always been
 an alias of put_page, so just use that instead.

--- a/nvidia/os-mlock.c
+++ b/nvidia/os-mlock.c
@@ -129,7 +129,7 @@
     else if (pinned < page_count)
     {
         for (i = 0; i < pinned; i++)
-            page_cache_release(user_pages[i]);
+            put_page(user_pages[i]);
         os_free_mem(user_pages);
         return NV_ERR_INVALID_ADDRESS;
     }
@@ -156,7 +156,7 @@
     {
         if (write)
             set_page_dirty_lock(user_pages[i]);
-        page_cache_release(user_pages[i]);
+        put_page(user_pages[i]);
     }
 
     os_free_mem(user_pages);
